client/src/Components/Location/Location.js

```diff
@@ -2,18 +2,45 @@ import React, { Component } from 'react';
 import './Location.css';
 
 class Location extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {wifi: false, privacy: false, quiet: false, foodAndDrink: false};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      [event.target.id]: !this.state[event.target.id]
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+  }
+
   render() {
     return (
-      <div>
-        <h1>Location</h1>
-        <div>This is a test.</div>
+      <div ref="outputDiv">
+        <form ref="form" onSubmit={this.handleSubmit}>
```

Comment:
I don't think we want a form tag here, because this won't be submitted separately.